samples/language/GettingStarted.qs

```diff
@@ -1,13 +0,0 @@
-/// # Sample
-/// Getting started
-///
-/// # Description
-/// This is a minimal Q# program that can be used to start writing Q# code.
-namespace MyQuantumProgram {
-
-    @EntryPoint()
-    operation Main() : Result[] {
-        // TODO: Write your Q# code here.
-        return [];
-    }
-}
```

Comment:
That's a great question, I hadn't considered it. The sample is still useful since we use it as the front page at https://microsoft.github.io/qsharp/ and it's nice to be able to just go in and start typing in a blank(-ish) template.

For now, I think it's ok to change it to the following:

```qsharp
@EntryPoint()
operation Main() : Unit {
    use q = Qubit();
}
```